runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -380,6 +380,21 @@ public XWalkWebChromeClient getXWalkWebChromeClient() {
         return mContentsClientBridge.getXWalkWebChromeClient();
     }
 
+    public int getContentHeight() {
+        /**
+         * When page finished loading, it's content size info may not be updated.
+         * Wait for a while for info updating.
+         */
+        if ((int)Math.ceil(mContentViewCore.getContentHeightCss()) == 0) {
+            try {
+                Thread.sleep(50);
```

Comment:
I consider this for some time. and I think it is not a good idea to hold UI thread on our side. I'll modify this and add comment to remind users.